file_formats/fip.md

```diff
@@ -116,6 +142,24 @@ def load_average_frame(video_file, start_frame, end_frame, frame_size, dtype, fr
     return (accumulated_frame / num_frames).astype(dtype)
 ```
 
+### Software Events
+
+Software events contain XXX(regions are already stored in csv and are not an event. What is the RepositoryStatus?) 
+
+### Metadata
+
+The fiber imaging system uses a single camera to capture data from two distinct light sources through temporal multiplexing. Because of that, only two metadata files are dropped; one for each camera. The temporally multiplexed channels are indicated in the file naming format below. These files can be used to QC the cameras.
+
+* `camera_green_iso_metadata.csv`: metadata from the camera recording from both green and iso channels
+* `camera_red_metadata.csv`: metadata from the camera recording from the red channel
+
+Within the metadata files are the following columns
+
+* `CameraFrameNumber` Camera frame number provided by camera hardware
+* `ReferenceTime` Camera reference time XXX more description here
+* `CameraFrameTime`  XXX
+* `CpuTime`  Software timestamp from the OS, in timezone (XXX)
```

Comment:
`CpuTime` here but `CPUTime` above.

Reply:
CpuTime is what Id prefer. I tend to stick with the [MS](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers) guidelines.

> The PascalCasing convention, used for all identifiers except parameter names, capitalizes the first character of each word (including acronyms over two letters in length), as shown in the following examples: